### ISSUE TITLE
Quiz creation: Display actual exercises within accordion panels

### ISSUE THREAD

**Issue by nucleogenesis** [learningequality/kolibri#11549]
## Overview
Fixing this depends on #11511  being merged so that we can add real content to a quiz section. The changes here only rely on having a quiz section with real exercises as questions so should present low risk of conflict and at worst light friction with changes to the underlying data architecture or other parts of the system
## Acceptance criteria
Given that a section has had questions added to it, the user should be able to expand the accordion item for the question and see a preview of the question rendered and visible similarly to how it was in the previous version of quiz creation.
Use the data architecture as-is so long as the `coach/assets/src/composables/quizCreationSpecs.js` object specs reflect the structures that you're getting and using to develop the interface.
- [ ] Reflect the [Figma](https://www.figma.com/file/wRlClsHgcVqIwgUrntRN1a/Kolibri-Enhanced-Quiz-Management?type=design&node-id=28-9053&mode=design&t=TF2JxLnbZ7LKj4kh-0)

**NOTE: Do not worry about any of the editing workflow present in the Figma, that is out of scope**

**Comment by nikkuAg** [comment]
Hey, is this issue still open? I would like to work on this

**Comment by MisRob** [comment]
Hi @nikkuAg, thanks for volunteering! I don't think this is open for contribution and also it seems to be blocked by some other work that's still in progress.

We currently have other contributing opportunities in three repositories. You can see the contributing guidelines including links to issues suitable for contribution for each repository here:

Kolibri: https://github.com/learningequality/kolibri/blob/release-v0.16.x/CONTRIBUTING.md
Kolibri Design System: https://github.com/learningequality/kolibri-design-system/blob/main/CONTRIBUTING.md
Studio: https://github.com/learningequality/studio/blob/unstable/CONTRIBUTING.md
